packages/http-client-csharp/eng/scripts/Generate.ps1

```diff
@@ -49,34 +50,46 @@ function IsSpecDir {
 }
 
 $failingSpecs = @(
-    Join-Path 'http' 'payload' 'pageable'
-    Join-Path 'http' 'payload' 'xml'
-    Join-Path 'http' 'special-headers' 'conditional-request'
-    Join-Path 'http' 'type' 'model' 'flatten'
-    Join-Path 'http' 'type' 'model' 'templated'
+    Join-Path 'specs' 'payload' 'pageable'
+    Join-Path 'specs' 'special-headers' 'conditional-request'
+    Join-Path 'specs' 'type' 'model' 'flatten'
+    Join-Path 'specs' 'type' 'model' 'templated'
+)
+
+$azureWhiteSpecs = @(
+    Join-Path 'specs' 'client' 'naming'
+    Join-Path 'specs' 'client' 'structure' 'client-operation-group'
+    Join-Path 'specs' 'client' 'structure' 'default'
+    Join-Path 'specs' 'client' 'structure' 'multi-client'
+    Join-Path 'specs' 'client' 'structure' 'renamed-operation'
+    Join-Path 'specs' 'client' 'structure' 'two-operation-group'
+    Join-Path 'specs' 'resiliency' 'srv-driven'
```

Comment:
I think this repo should not contain those azure specs.
They are not generated previously therefore they should be generated or tested now.

Reply:
They are generated before.

Reply:
if this is true, this means the new cadl ranch is missing some specs. They should add them.
this is not a fair reason that we add them from the azure spec here

Reply:
it is OK that we introduce this in this PR, but please create an issue to track this - this is abnormal.

Reply:
>  this means the new cadl ranch is missing some specs. 

I'm not sure if it is correct. New cadl-ranch tests are split into two categories as I put in the description. No matter we are azure generator or standard generator, we could decide we generate tests from azure cadl-ranch or standard cadl-ranch or both. In this PR I choose to generate tests from both. That is why you could see that I didn't delete any tests in this PR.

As to how a specific test fits into one of these two categories, I don't care in this PR. I don't even care later form the generator perspective. It is cadl-ranch's responsibility.

Reply:
BTW, new azure cadl-ranch + new standaredcadl-ranch don't missing any specs.

Reply:
You are saying the same thing as I have in this comment.
This generator is an unbranded generator, therefore it should only run against the general typespec specs. The azure specs should not be included.
We need to create an issue to track this if you believe this issue is a new cadl ranch issue, then it is fine that we introduce them here, and when something changed in that part, we should adopt later.

The point is that when we are doing something, we find something abnormal, instead of ignoring them, we should find the root cause of it (if possible) and create issues to track them and move on in the current PR.

Reply:
https://github.com/microsoft/typespec/issues/5400